### PR TITLE
Update portal using fluent ui Link component for translations string

### PR DESCRIPTION
**Pull request by carmenlau**
Otherwise the link style is missing

### CODE REVIEW

**Comment by louischan-oursky** [comment]
Actually we still have a problem. If we use the bare Link from fluent, when the link is clicked, the app is reloaded.

To avoid reloading, we must use the Link from react-router-dom. I have built `src/ReactRouterLink.tsx` to make it work with the Link from fluent as we expect. You can read the source code of it to know the reason why it is there.

Finally, what we want here is to make [react-messageformat](https://github.com/oursky/react-messageformat) to support default `components`, so we do not need to provide `components` everywhere.

I suggest we add the default `components` in react-messageformat and then configure it to have a default `Link` component, which is backed by `ReactRouterLink`

**Comment by carmenlau** [comment]
Updated using latest `react-messageformat`

File: portal/src/ReactApp.tsx [hunk 1]
```diff
@@ -112,7 +119,14 @@ const ReactApp: React.FC = function ReactApp() {
...
   if (error != null) {
     return (
-      <LocaleProvider locale="en" messageByID={MESSAGES}>
+      <LocaleProvider
+        locale="en"
+        messageByID={MESSAGES}
+        defaultComponents={{
+          ReactRouterLink: PortalLink,
+          Link: FluentLink,
+        }}
```
Comment (louischan-oursky) [inline portal/src/ReactApp.tsx:128]: The `defaultComponents` can be defined as static variable.

**Comment by louischan-oursky** [comment]
Some of the usage of `Link` in `en.json` is missing `target="_blank"` and `rel="noreferrer"`. Can we add them back?

**Comment by carmenlau** [comment]
Updated, see if it looks good!:)

**Pull request merged**